Readme upgrade (#10)
UXPin/uxpin-merge-boilerplate-proptypes: README.md

All the components are built with React.js and styled with Emotion. Styles are using tokens stored in JSON files.

To learn more about Merge – visit [our wiki](https://wikiuxpin.atlassian.net/wiki/spaces/MA/overview)

> ## Installation

Reviewer: I think the `## Tests` section is in a wrong place – I'd put it at the bottom of the readme, or at least under "Adding new components", as it's a detail important for repository maintainers, not a users experimenting with the technology.

* You have to add the component to uxpin.config.js file

> * You have to prepare presets for every component (temporary restrictions to be replaced by jsx presets)

Reviewer: This point should be corrected.
```suggestion
 * You have to prepare a default preset for every component
```

Every component has to be referenced in UXPin Merge Config file [`./uxpin.config.js`](https://wikiuxpin.atlassian.net/wiki/spaces/MA/pages/665714725/UXPin+Merge+Config+File).

> ## Authoring JSX Presets

Reviewer: I think it's a bad idea to duplicate here the documentation of authoring the presets on CI configuration. IMHO we should put here only a link to the wiki documentation, otherwise it may be really hard to maintain the same documentation in different places.
